packages/gatsby-theme-styleguide/src/gatsby-components/SiteMapData/tools.js

```diff
@@ -16,7 +16,7 @@ export const parseAllFiles = (allMdx, { basedir = 'docs/', location } = {}) => {
     pathArray.pop()
     const level = pathArray.length
 
-    const { title, order } = mdxElem.frontmatter
+    const { title, order, description, image } = mdxElem.frontmatter
```

Comment:
We don't get "image" from frontmatter anymore. The presence of the preview.png is detected automatically.